packages/redux/src/sharedWishlists/selectors.ts

```diff
@@ -154,22 +157,96 @@ export const getSharedWishlistItem = (
  * ```
  *
  * @param state     - Application state.
- * @param sharedWishlistId - Numeric identifier of the shared wishlist.
```

Comment:
Should this be removed?

Reply:
Yes, my bad. At first I removed it because I was thinking of doing another approach, but then I changed again and forgot to add the jsdoc :(
Fixed, thanks!